lib/versioning/VersioningRequestProcessor.js

```diff
@@ -255,19 +255,13 @@ class VersioningRequestProcessor {
         // - versioning && !versionId: create a new version
         // - !versioning && versionId: update (PUT/DELETE) an existing version
         //                             if versionId === '' update master version
-        // other cases are invalid and metadata does nothing
+        // - versioning && versionId: create a new version which has
+        //   the specified versionId (e.g. for async replication)
+
         const versioning = options &&
             (options.versioning || options.versioning === '');
```

Comment:
I know this is from before but if we expect versioning to be true/false, why do we interpret `versioning: ''` as `versioning: true`?